bio/ragtag/merge/wrapper.py

```diff
@@ -0,0 +1,55 @@
+"""Snakemake wrapper for ragtag-merge."""
+
+__author__ = "Curro Campuzano Jiménez"
+__copyright__ = "Copyright 2023, Curro Campuzano Jiménez"
+__email__ = "campuzanocurro@gmail.com"
+__license__ = "MIT"
+
+
+from snakemake.shell import shell
+import tempfile
+
+

```

Comment:
I just noticed that the wrapper uses `snakemake.params.extra` directly. This will fail if the rule does no define `params.extra`. To avoid that, it is better to check and `extra`:

```suggestion
extra = snakemake.params.get("extra", "")
```

And can you also change the other wrappers?